### ISSUE TITLE
After login nothing happen.

### ISSUE THREAD

**Issue by falcom0** [feisalfahmi/mobile_project#1]
In login_screen.dart after submit or after authenticated with email and password there is no route to HomeScreen

**Comment by falcom0** [comment]
yes indeed, i was registered as user in your apps. but after I attempted to login to your apps, nothing happens. because you did not route after it was submitted to firebase. I saw your code. check your codes again.

**Comment by feisalfahmi** [comment]
Astaghfirullah, sorry sir, we initialized it on main.dart. If "id" already exists it will go to the homescreen.dart. When logging in, we close the application, then we open it, it will go to homescreen. This is a mistake in our coding 🙏🙏
